Gemfile

```diff
@@ -49,9 +40,7 @@ group :development do
 end
 
 group :development, :test do
-  gem 'awesome_print'
   gem 'bullet'
-  gem 'bundler-audit', '>= 0.5.0', require: false
```

Comment:
I believe this is the gem that @monfresh uses to upgrade our gems? So it's not used in the app, but it's used on the app?